apps/factory/src/services/vendors/openai_vendors.ts

```diff
@@ -45,6 +46,7 @@ class OpenAIVendor extends BaseVendor {
         {
           index: 0,
           text: response.choices[0]?.message.content,
+          tool_calls: response.choices[0]?.message?.tool_calls,
           logprobs: null,
           finish_reason: "stop",
```

Comment:
@pulkit21 this reason need to fill from response. with tools, this will have different response